Makefile

```diff
@@ -199,6 +199,12 @@ linux32: jni-header
 linux64: jni-header
 	docker run $(DOCKER_RUN_OPTS) -i -v $$PWD:/work xerial/centos5-linux-x86_64-pic bash -c 'make clean-native native-nocmake OS_NAME=Linux OS_ARCH=x86_64'
 
+alpine-image:
+	docker build -f docker/Dockerfile.alpine -t snappy-java-alpine .
+
+alpine: jni-header
+	docker run $(DOCKER_RUN_OPTS) -i -v $$PWD:/work snappy-java-alpine bash -c 'make clean-native native-nocmake OS_NAME=Alpine OS_ARCH=x86_64'
```

Comment:
snappy-java uses OSInfo for resolving OS_NAME (os.name Java system property) https://github.com/xerial/snappy-java/blob/266126036c1784825e3e470ba2b27744dee6b13c/src/main/java/org/xerial/snappy/OSInfo.java#L129-L131

Does Alpine Linux return os.name=Alpine?

Reply:
It returns Linux instead. I wasn’t aware of this issue. Do you have any suggestion to fix this?

```bash
> docker run -it openjdk:17-alpine sh
/ # vi CheckOSName.java
/ # javac CheckOSName.java
/ # java CheckOSName
Linux

Reply:
If so, we need some change to OSInfo.java to detect the platform is Alpine or not.

My preference is like this:
- getOSName() = Linux
- getArchName() = x86_64-alpine 

And change the makefile to use these names instead

Reply:
Already fixed that, can you take another look? I also choose to use the name getArchName() = x86_64-musl, since it reflects better the architecture and can possible be used by other systems using musl besides alpine